src/main/play-doc/operation/DynamicProxyConfiguration.md

```diff
@@ -556,6 +556,77 @@ Here are the explanation for the example template above:
+
+```
+#!/bin/bash
+
+# Note - this script is for testing purposes only.
+
+set -e
+
+SCRIPT_DIR="$( cd "$(dirname $0)" && pwd)"
+CERT_NAME=ssl-"conductr.cli"
+SSL_CERT_DIR="$SCRIPT_DIR/$CERT_NAME"
+
+#rm $SSL_CERT_DIR/*
+mkdir -p $SSL_CERT_DIR
+
+echo "Generating SSL certs and keys into $SSL_CERT_DIR"
+openssl genrsa -out $SSL_CERT_DIR/$CERT_NAME.key 2048
+#This line creates the certificate - Most important fields are CN=.. and subjectAltName=... These are the domains that the cert can be used with. For the purposes of the ConductR CLI, this must cover the Cores at least.
+#For Dev servers, this would likely be (untested): *.ebiz.verizon.com
+openssl req -new -key $SSL_CERT_DIR/$CERT_NAME.key -sha256 -nodes -subj '/C=US/ST=California/L=San Francisco/O=Lightbend/OU=Prod Suite/CN=192.168.99.100/emailAddress=dev.null@lightbend.com' -out $SSL_CERT_DIR/$CERT_NAME.csr
+openssl x509 -req -days 365 -in $SSL_CERT_DIR/$CERT_NAME.csr -signkey $SSL_CERT_DIR/$CERT_NAME.key -out $SSL_CERT_DIR/$CERT_NAME.crt
+
+echo "Concat crt and key into pem file"
+cat $SSL_CERT_DIR/$CERT_NAME.crt $SSL_CERT_DIR/$CERT_NAME.key | tee $SSL_CERT_DIR/$CERT_NAME.pem
+```
+
+HAProxy is then configured with the path of our installed certificate and credentials to apply Basic Authentication to incoming ConductR Control API requests via port `9005`. To do this, we create a frontend `ctrl_frontend_secure` to handle incoming Control API requests, a userlist `ctrl_users` and a backend `ctrl_backend_auth` to direct to one of three available Conductr Cores, running on `172.17.0.1`,`172.17.0.2` and `172.17.0.3` listening to port `9055` as configured above.
+
+```
+frontend dummy
+  bind 127.0.0.1:65535
+
+frontend ctrl_frontend_secure
+  bind 0.0.0.0:9005 ssl crt /path/to/my/sslcert/ssl-conductr.cli.pem
+  mode http
+  acl ctrl_context path_beg /
+  use_backend ctrl_backend_auth if ctrl_context
+
+userlist ctrl_users
+  user steve insecure-password stevespassword
+
+backend ctrl_backend_auth
+  mode http
+  server ctrl_server_1 172.17.0.1:9055 maxconn 1024
+  server ctrl_server_2 172.17.0.2:9055 maxconn 1024
+  server ctrl_server_3 172.17.0.3:9055 maxconn 1024
+  acl ctrl_auth_ok http_auth(ctrl_users)
+  http-request auth realm ConductR if !ctrl_auth_ok
+```
+
+With the above setup, Control Protocol requests must be sent via HTTPS with Basic Authentication credentials. The ConductR CLI supports this via a credentials file that must created at the following location `~/.conductr/settings.conf`, which must point to PEM file used to validate the SSL certificate locally (ss-conductr-cli.pem in the above example). Configuring this file is decribed in detail the ConductR CLI documentation section called [HTTP Basic Authentication](CLI#HTTP-Basic-Authentication).
```

Comment:
Grammar corrections:
- `credentials file that must be created` instead of `credentials file that must created`
- `Configuring this file is described` instead of `Configuring this file is decribed`